ui/index.d.ts

```diff
@@ -1068,7 +1069,7 @@ export interface IStorageAccountWizardContext extends IResourceGroupWizardContex
      * If an existing storage account is picked, this value will be defined after `StorageAccountListStep.prompt` occurs
      * If a new storage account is picked, this value will be defined after the `execute` phase of the 'create' subwizard
      */
-    storageAccount?: StorageManagementModels.StorageAccount;
```

Comment:
As mentioned [here](https://github.com/microsoft/vscode-azurestorage/pull/882#discussion_r527160916), I don't think we need to actually reference the types for the hybrid npm package